app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -1,5 +1,9 @@
 module ManageIQ::Providers::Kubernetes::ContainerManager::EventCatcherMixin
   extend ActiveSupport::Concern
+  ENABLED_EVENTS = {
+    'Node' => %w(NodeReady NodeNotReady rebooted),
+    'Pod'  => %w(scheduled failedScheduling failedValidation HostPortConflict created failed started killing)
```

Comment:
I would try to correctly categorize the events on our side. If I am not mistaken it should be:

    'Pod'       => %w(scheduled failedScheduling failedValidation HostPortConflict)
    'Container' => %w(created failed started killing)

(Please double check that)